src/azure-cli/azure/cli/command_modules/cosmosdb/_help.py

```diff
@@ -204,3 +204,437 @@
     text: az cosmosdb keys list --name MyCosmosDBDatabaseAccount --resource-group MyResourceGroup --subscription MySubscription
     crafted: true
 """
+
+helps['cosmosdb sql'] = """
+type: group
+short-summary: Manage Azure Cosmos DB SQL resources.
```

Comment:
You can consider something like "Manage SQL resources of Azure Cosmos DB account." The same as other resource, e.g. mongodb.